spec/features/admin_deletes_a_review_spec.rb

```diff
@@ -0,0 +1,78 @@
+require 'rails_helper'
+
+feature "Admin deletes review" do
+
+  scenario 'I want to delete a review and am not an admin' do
+
+    user = FactoryGirl.create(:user)
+
+    visit new_user_session_path
+
+    fill_in 'Email', with: user.email
+    fill_in 'Password', with: user.password
+
+    click_button 'Log in'
+    user.admin = false
+
+    spot = Spot.create!(
+      name: 'asdfasdf',
+      description: 'lala',
+      category: 'asdf',
+      address: '44 street',
+      city: 'Dove',
+      state: 'MA',
+      zip_code: '12345',
+      phone: '1234565432',
+      user: user
+    )
```

Comment:
I would recommend making factories for Spot and Review and using those - this test is unnecessarily long due to how much info you need to put into the creation process here!